api/diagnostics.go

```diff
@@ -484,14 +484,13 @@ func (d diagnosticsJobCanceledError) Error() string {
 }
 
 // fetch an HTTP endpoint and append the output to a zip file.
-func (j *DiagnosticsJob) getHTTPAddToZip(node Node, endpoints map[string]string, folder string, zipWriter *zip.Writer,
```

Comment:
How did this ever build? I would expect errors on the unused parameter...

Reply:
Good point. The check was disabled. I fixed it here https://github.com/dcos/dcos-diagnostics/pull/62